tests/Old/CommonExtractorTest.php

```diff
@@ -604,7 +609,7 @@ public function testTableColumnsQuery(): void
         $outputTableName = $result['imported'][0]['outputTable'];
         $this->assertExtractedData($this->dataDir . '/simple.csv', $outputTableName);
         $manifest = json_decode(
-            (string) file_get_contents(sprintf("%s/out/tables/%s.csv.manifest", $this->dataDir, $outputTableName)),
+            (string) file_get_contents($this->dataDir . '/out/tables/' . $outputTableName . ".csv.manifest"),
```

Comment:
mixed quotes

Reply:
Nejaky mistake po rebasu

Reply:
Alias
`dft = !git diff `git rev-parse --symbolic-full-name --abbrev-ref=strict HEAD@{u}`..HEAD`

Porovná ti branch s tím, co bylo před rebasem v upstreamu. Takže když uděláš "git push; git rebase -i origin/master; git dft" tak uvidíš změny, co se objevili v té branchi. Pokud uděláš jen rebase "na sebe", tak to funguje ještě líp a vidíš jen co se změnilo v té branchi (a ne co ti tam nového přinesl nový master).